### DISCUSSION TITLE
How to remove the forked Repository ?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#27586] [Code Security]
I created a Repository called "swipe"  in "freerfing" personally last year,which contained some codes and http urls are owned by  some organizations.Later found that before we deleted, the user named xuxioajin123 completely forked our entire Repository, and it still exists.
   The following is Github's URL and Screenshots about our forked Respository: 
**https://github.com/xuxioajin123/swipe**
   Since "xuxioajin123/swipe" contains some codes and http urls are owned by  some organizations, so we hope Github as soon as possible to help me completely remove  "xuxioajin123/swipe"  and thank you very much!

**Comment by jfullstackdev** [comment]
you mean you created a repo that uses code from other projects, then another forked that repo, but you are concerned with copyright because of other licenses contained in your repo ? and if you delete your own repo, the fork will still exist ? 

but it's not a fork, say, they made a standalone out of that, but you have commits in there 

https://github.com/xuxioajin123/swipe/commits/main

![image](https://user-images.githubusercontent.com/47092464/181741291-d59f67eb-cbab-438f-b6b9-fec3989f830a.png)

**Comment by airtower-luna** [comment]
The [GitHub ToS](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#5-license-grant-to-other-users) explicitly state:

> By setting your repositories to be viewed publicly, you agree to allow others to view and "fork" your repositories (this means that others may make their own copies of Content from your repositories in repositories they control).

So the user is allowed to have their fork, and I think your best option is to politely ask them to delete it.
